docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md

```diff
@@ -0,0 +1,199 @@
+# Getting started with Mambaforge and Python 
+[Mara Lampert](../readme), January 26th, 2023
+
+## Introduction to Python and Mamba 
+This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/installation.html)/ Mambaforge, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
+
+> **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/johannes_mueller/anaconda_getting_started/) written by Johannes. 
+
+
+**Why do we need Mamba to use Python?**
+
+__Python__ is a programming language which is easy to learn and very important in scientific data analysis nowadays. 
+
+__Mamba__ is a __package manager__ which can be used with Python. It is a software allowing to install other software. Read more about Mamba [here](https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/). 
+
+## Installation of Mambaforge 
```

Comment:
One thing I was wondering: If I have another distribution already installed (e.g., Anaconda), do I have to uninstall it? Can I keep existing environments? Most of them I obviously don't use anymore but some I'd be a bit reluctant to delete if I'd have to ^^

Spoiler: I had anaconda installed previously and can still use my old environments.